### PR TITLE
support own your storage for video

### PR DESCRIPTION
**Pull request by MoonsuCha**
## Change description

Support own your storage for video

## Test & Review

- aws, gcp, azure test success

### CODE REVIEW

File: spb_label/core/session.py [hunk 1]
```diff
@@ -174,12 +174,15 @@ def execute(self, query: str, values: dict = {}):
             self.history.appendleft({"APIException": data})
             raise APIException(f"Network Connection Error: {repr(e)}")
         except Exception as e:
+            print(e)
             self.history.appendleft({"APIException": data})
+            print("sss")
```
Comment (Min-june) [inline spb_label/core/session.py:179]: 삭제

File: spb_label/core/session.py [hunk 2]
```diff
@@ -174,12 +174,15 @@ def execute(self, query: str, values: dict = {}):
...
             raise APIUnknownException(
                 "Unknown Error Exception: Check your API response"
             )
 
         result = response.json()
+        print(result)
```
Comment (Min-june) [inline spb_label/core/session.py:185]: 삭제

**Review by Min-june — APPROVED** [review]

File: spb_label/core/session.py (hunk 1 above)
Comment (MoonsuCha) [reply spb_label/core/session.py:179]: Thanks!

File: spb_label/core/session.py (hunk 2 above)
Comment (MoonsuCha) [reply spb_label/core/session.py:185]: Thanks!

**Pull request merged**